site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -73,18 +75,16 @@ Start by creating a file named `background.js` inside the extension's directory.
 }
 ```
 
-Background scripts, like many other important components, must be registered in the manifest.
-Registering a background script in the manifest tells the extension which file to reference, and how
-that file should behave.
-
 Chrome is now aware that the extension includes a service worker. When you reload the extension,
 Chrome will scan the specified file for additional instructions, such as important events it needs
 to listen for.
 
+### Create the background script {: #background_script }
+
 This extension will need information from a persistent variable as soon as it's installed. Start by
 including a listening event for [`runtime.onInstalled`][11] in the background script. Inside the
 `onInstalled` listener, the extension will set a value using the [storage][12] API. This will allow
-multiple extension components to access that value and update it.
+multiple extension components to access that value and update it. Inside the extension's directory create a file named `background.js` and add the following code.
 
 ```js
 let color = '#3aa757';
```

Comment:
We talked about also adding a comment like `// background.js` and I think it would be good. I don't think it's overkill to add both the "create a file..." sentence above and the code comment.

Reply:
Done